content/docs/getting-started/install-github-app/index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Installing the GitHub App
+publishedDate: '2022-03-21T21:00:00.0Z'
+description: How to give Roadie the access it needs to your GitHub org.
+---
+
+## Introduction
+
+This tutorial will guide you through the steps required to connect Roadie to your GitHub org.
+
+## Prerequisites
+
+If your IT department filters your outbound access by hostname, you may need to allow access to Roadie hostnames before proceeding. More information on this can be found [here](/docs/details/allowlisting-roadie-traffic/).
```

Comment:
I think the allow list page doesn't list hostnames, just our IPs. We should add those hostnames in there as well though.

Reply:
Ok. I don't know what the hell I did in https://github.com/RoadieHQ/marketing-site/pull/625 but somehow I removed the hostnames completely from the allowlist docs.

Reply:
This PR doesn't cause that problem though. So can I merge this one and fix the allowlist in a separate PR today?

Reply:
:+1:

Reply:
I was adding more to this PR to incorporate Brian's suggestions so I made this change while I was at it.